Translation/en/RedirectedResources/assets/abdata/adv/scenario/op/00/0/translation.txt

```diff
@@ -3,75 +3,75 @@
 「あなたは、先ほどのフェリーに乗ってたお客様ですよね？」=“You're the customer who just got off the ferry, right?”
 「あ、そうです。俺の名前は――」=“Ah, yeah, that's me. My name is――”
 「俺の名前は[P名前]って言います。えっと、よろしくお願いします」=“My name is [P名前]. Uhh, it's nice to meet you.”
-「はい、よろしくお願いします！　早速ですが、何か困ってないですか？　私に出来る事なら何でも言ってください！」=“Yes, nice to meet you too! To get to the point, is something bothering you? Please tell me if you need something!”
+「はい、よろしくお願いします！　早速ですが、何か困ってないですか？　私に出来る事なら何でも言ってください！」=“Yes, nice to meet you too! Firstly, are you having any troubles? Please tell me if you need something!”
 途方に暮れていた俺にとって、彼女の笑顔は太陽のように輝いて見えた。=Even though I was at a loss, her smile shined like the sun.
-「ありがとうございます。実はよく分からないまま、この島に来ることになってしまって……正直、全てにおいて困っていたんです」=“Thank you very much. To be honest, I came to this island without really knowing much about it... so frankly, I'm bothered by pretty much everything.”
+「ありがとうございます。実はよく分からないまま、この島に来ることになってしまって……正直、全てにおいて困っていたんです」=“Thank you very much. To be honest, I came to this island without really knowing much about it... so frankly, I'm having trouble with everything.”
 「あはは……それならまず、この島について簡単に説明しますね」=“Ahaha... then if that's it, then I'll start by briefly explaining about this island.”
-「ここは、年間を通して多くの方にお越しいただいている観光地になってます」=“This place is a tourist destination that is visited by a lot of people throughout the year.”
-「一番人気は、なんといっても島を囲う綺麗な海！　ただ眺めるのもよし、様々なレジャーで遊ぶのもよし、楽しみ方はお客様次第です」=“The most popular place is the beautiful sea that surrounds the island! You can just watch it, or play all sorts of leisure activities... how you enjoy yourself is up to you!”
-「他にも自然豊かな公園や、カップルに人気のスポットがある灯台、たくさんのお魚が泳いている水族館などなど、色んな名所があるんです」=“Furthermore, there's lots of natural parks, and a popular spot for couples is the lighthouse, and there's an aquarium with all sorts of fish swimming, so we have a lot of attractions.”
-（なるほど……コイカツをするのにピッタリの島だ。だから叔母さんもここを選んだんだろうな）=(I get it... this is precisely the sort of island for koikatsu. So that's probably why my aunt chose this place.)
-「素敵なところなんですね。説明、すごく分かりやすかったです」=“It seems like a nice place. Your explanation shows you know an awful lot about it.”
-「えへ……私、この島出身なので、褒めてもらえて嬉しいですっ」=“Ehe... I'm a native of this island, so I'm glad for your praise.”
-「って、喜んでる場合じゃなかった……えっと、他に何か聞きたいことはないですか？」=“But well, if that still hasn't satisfied you... uhh, is there anything else you'd like to ask?”
+「ここは、年間を通して多くの方にお越しいただいている観光地になってます」=“This is a popular tourist destination that has lots of visitors throughout the year.”
+「一番人気は、なんといっても島を囲う綺麗な海！　ただ眺めるのもよし、様々なレジャーで遊ぶのもよし、楽しみ方はお客様次第です」=“The most famous feature is the beautiful sea that surrounds the island! You can just enjoy the view, or engage in a variety of leisure activities... It's up to you!”
+「他にも自然豊かな公園や、カップルに人気のスポットがある灯台、たくさんのお魚が泳いている水族館などなど、色んな名所があるんです」=“There's also our nature park, a lighthouse (a popular spot for couples), an aquarium filled with fish, and many other attractions!”
```

Comment:
Any reasons why (a popular spot for couples) is in parentheses? Also should be "the lighthouse" since there's only one on the island.

Reply:
True, parentheses don't make sense in spoken lines. Maybe change it to “There's also our nature park, the couples' favorite lighthouse, the fish-full aquarium, and many other attractions!”

Reply:
Because it reads as if spoken as an aside, which would normally be offset via commas, but commas are splitting the enumeration, but how about this:

```suggestion
「他にも自然豊かな公園や、カップルに人気のスポットがある灯台、たくさんのお魚が泳いている水族館などなど、色んな名所があるんです」=“There's also our nature park, the lighthouse which is a popular spot for couples, an aquarium filled with fish, and many other attractions!”
```